LanguageFeatures/Patterns/pattern_variable_sets_A01_t01.dart

```diff
@@ -105,13 +100,8 @@ String test2(Object x) {
 // [analyzer] unspecified
 // [cfe] unspecified
     return "variable";
-  if (x case var _ || final _)
-//                    ^^^^^
-// [analyzer] unspecified
-// [cfe] unspecified
-    return "variable-2";
   if (x case final num? v6 || (var num? v6) when v6! == 6)
```

Comment:
By the way, I think this would be a syntax error, because `var T id` is never supported. I guess `case final num? v6 || (num? v6)` would do?